examples/using-module/index.html

```diff
@@ -5,7 +5,9 @@
 </head>
 
 <body>
-    <div id='map'></div>
+    <!-- <div id='container'> -->
```

Comment:
do we need this commented code? what's it for?

Reply:
Ah, nice catch. I was modding the existing examples instead of wasting time making new ones. I'll remove it.